x-pack/plugins/maps/public/connected_components/layer_panel/layer_settings/layer_settings.tsx

```diff
@@ -66,6 +73,32 @@ export function LayerSettings(props: Props) {
     }
   };
 
+  const renderIncludeInFitToBounds = () => {
+    if (!props.supportsFitToBounds) {
+      return null;
+    }
+    return (
+      <EuiFormRow display="columnCompressedSwitch">
+        <EuiToolTip
+          position="top"
+          content={i18n.translate('xpack.maps.layerPanel.settingsPanel.fittableFlagTooltip', {
+            defaultMessage: `Fit to data bounds adjusts your map extent to show all of your data. Layers that provide reference data may not need to be included in the data bounds. Use this option to exclude a layer from the fit to data bounds computation.`,
+          })}
+        >
+          <EuiSwitch
+            label={i18n.translate('xpack.maps.layerPanel.settingsPanel.fittableFlagLabel', {
+              defaultMessage: `Include bounds in fit to data bounds computation`,
```

Comment:
Nit: How about replacing "bounds" with "layer" or just removing it so it becomes "Include layer in fit to data bounds computation" or just "Include in fit to data bounds computation". I think there is a gap and users have to make a leap that "bounds" is the bounds for this layer. I think either "layer" or no noun is easier to understand.